spec/default_spec.rb

```diff
@@ -48,6 +49,7 @@
     it 'should config JAAS' do
       chef_run.converge(described_recipe)
       expect(chef_run).to create_template('/opt/confluent/confluent-2.0.1/jaas.conf')
+      # rubocop:disable Lint/AmbiguousBlockAssociation
```

Comment:
Actually had to revert this change as then rspec failed with `Must specify expected content or a block!` https://travis-ci.org/bbaugher/confluent/builds/236086089